eng/pipelines/apiview-review-gen-cadl.yml

```diff
@@ -35,13 +35,13 @@ jobs:
       Write-Host "Reviews: $($reviews)"
       echo "##vso[task.setvariable variable=Reviews]$reviews"
     displayName: "Setup Reviews Variable"
-    condition: eq(variables['Reviews'], '')
+    condition: and(succeeded(), eq(variables['Reviews'], ''))
```

Comment:
I don't think we need this pipeline file anymore. CADL was added a long back to support it when it was named as CADL before TypeSpec.